src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/HelloWorldControllerTest.java

```diff
@@ -31,4 +32,22 @@ void shouldSayHelloWorld() throws Exception {
         // then
         assertEquals("Hello World!", result);
     }
+
```

Comment:
I have to discuss it with other leaders, but I think that tests have to be made in given-when-then structure.